src/utils/ipfs.ts

```diff
@@ -54,11 +54,29 @@ export const logoNameForHandle = (handle: string) => `juicebox-@${handle}-logo`
 export const metadataNameForHandle = (handle: string) =>
   `juicebox-@${handle}-metadata`
 
-export const ipfsCidUrl = (hash: string) =>
-  `https://${IPFS_GATEWAY_HOSTNAME}/ipfs/${hash}`
+export const ipfsCidUrl = (hash: string, useFallback = false) => {
```

Comment:
**suggestion** for readability purposes, miight be better to have an object prop that includes `useFallback`.

That way instead of `ipfsCidUrl(hash, true)` (i dont know what `true` means without looking at the function internals), it would read `ipfsCidUrl(hash, { useFallback: true })

Reply:
too much python

will do